_docs/concepts/namespaces.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: guide
+title: StorageOS Docs - Namespaces
+anchor: concepts
+module: concepts/namespaces
+---
+
+# Namespaces
+
+StorageOS namespaces are an identical concept to Kubernetes namespaces. They
+are intended to allow a StorageOS cluster to be used by multiple teams across
+multiple projects. StorageOS maps Kubernetes namespaces 1 to 1 when
+PersistentVolumeClaims using the StorageOS StorageClass are created. 
+
```

Comment:
I'd add that there is no need to create them by hand as people might create a PVC in a NS go to the UI and not find the volume as you need to select the Namespace there.

--> StorageOS Namespaces are automatically created when a PVC on a NS in K8S is provisioned.